week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +43,8 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  const removed = arr.splice(index, 1); // complete this statement
+  return arr;
 }
```

Comment:
it's nearly flawless, except this bit: `NOT change the original array`
Read carefully what is different between `splice` (you used) and `slice`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Hint: If you use array methods, try to use `slice` and `concat` and then `slice` again. Otherwise, use for-loop to build a new array excluding a removed one.